### ISSUE TITLE
Trouble installing vagrant-libvirt plugin on MacOSX due to ruby-libvirt gem

### ISSUE THREAD

**Issue by jcsalem** [pradels/vagrant-libvirt#445]
While we've had good success using vagrant-libvirt on MacOSX, one of my coworkers ran into problems installing on his machine that I haven't been able to fix.
He's running Xcode 6.2 and Vagrant 0.7.4 on MacOSX Mavericks.
Running `vagrant plugin install vagrant-libvirt` gives the error below when trying to install the ruby-libvirt gem.  However, that gem installs fine if I do a `gem install ruby-libvirt -v '0.5.2'`

Any ideas?
Here's the (long) error message:

```
An error occurred while installing ruby-libvirt (0.5.2), and Bundler cannot continue.
Make sure that `gem install ruby-libvirt -v '0.5.2'` succeeds before bundling.

Gem::Installer::ExtensionBuildError: ERROR: Failed to build gem native extension.

make "DESTDIR="
compiling _libvirt.c
compiling common.c
In file included from common.c:27:
/opt/vagrant/embedded/include/ruby-2.0.0/ruby/backward/st.h:2:2: warning: #warning use "ruby/st.h" instead of bare "st.h"
In file included from common.c:27:
/opt/vagrant/embedded/include/ruby-2.0.0/ruby/backward/st.h:2:2: warning: #warning use "ruby/st.h" instead of bare "st.h"
compiling connect.c
compiling domain.c
In file included from domain.c:26:
/opt/vagrant/embedded/include/ruby-2.0.0/ruby/backward/st.h:2:2: warning: #warning use "ruby/st.h" instead of bare "st.h"
domain.c: In function ‘ruby_libvirt_domain_init’:
domain.c:5439: error: ‘VIR_DOMAIN_QEMU_AGENT_COMMAND_BLOCK’ undeclared (first use in this function)
domain.c:5439: error: (Each undeclared identifier is reported only once
domain.c:5439: error: for each function it appears in.)
domain.c:5443: error: ‘VIR_DOMAIN_QEMU_AGENT_COMMAND_DEFAULT’ undeclared (first use in this function)
domain.c:5447: error: ‘VIR_DOMAIN_QEMU_AGENT_COMMAND_NOWAIT’ undeclared (first use in this function)
domain.c:5451: error: ‘VIR_DOMAIN_QEMU_MONITOR_COMMAND_DEFAULT’ undeclared (first use in this function)
domain.c:5455: error: ‘VIR_DOMAIN_QEMU_MONITOR_COMMAND_HMP’ undeclared (first use in this function)
In file included from domain.c:26:
/opt/vagrant/embedded/include/ruby-2.0.0/ruby/backward/st.h:2:2: warning: #warning use "ruby/st.h" instead of bare "st.h"
domain.c: In function ‘ruby_libvirt_domain_init’:
domain.c:5439: error: ‘VIR_DOMAIN_QEMU_AGENT_COMMAND_BLOCK’ undeclared (first use in this function)
domain.c:5439: error: (Each undeclared identifier is reported only once
domain.c:5439: error: for each function it appears in.)
domain.c:5443: error: ‘VIR_DOMAIN_QEMU_AGENT_COMMAND_DEFAULT’ undeclared (first use in this function)
domain.c:5447: error: ‘VIR_DOMAIN_QEMU_AGENT_COMMAND_NOWAIT’ undeclared (first use in this function)
domain.c:5451: error: ‘VIR_DOMAIN_QEMU_MONITOR_COMMAND_DEFAULT’ undeclared (first use in this function)
domain.c:5455: error: ‘VIR_DOMAIN_QEMU_MONITOR_COMMAND_HMP’ undeclared (first use in this function)
{standard input}:4175:non-relocatable subtraction expression, "L_e_RetrieveError$non_lazy_ptr" minus "L00000000053$pb"
{standard input}:4175:symbol: "L_e_RetrieveError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:4102:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000052$pb"
{standard input}:4102:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:3935:non-relocatable subtraction expression, "_c_domain_job_info" minus "L00000000051$pb"
{standard input}:3935:symbol: "_c_domain_job_info" can't be undefined in a subtraction expression
{standard input}:3923:non-relocatable subtraction expression, "L_e_RetrieveError$non_lazy_ptr" minus "L00000000051$pb"
{standard input}:3923:symbol: "L_e_RetrieveError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:3851:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000050$pb"
{standard input}:3851:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:3802:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000049$pb"
{standard input}:3802:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:3743:non-relocatable subtraction expression, "L_ruby_libvirt_str_new2_wrap$non_lazy_ptr" minus "L00000000048$pb"
{standard input}:3743:symbol: "L_ruby_libvirt_str_new2_wrap$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:3725:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000048$pb"
{standard input}:3725:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:3665:non-relocatable subtraction expression, "L_e_RetrieveError$non_lazy_ptr" minus "L00000000047$pb"
{standard input}:3665:symbol: "L_e_RetrieveError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:3609:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000046$pb"
{standard input}:3609:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:3544:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000045$pb"
{standard input}:3544:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:3483:non-relocatable subtraction expression, "L_e_RetrieveError$non_lazy_ptr" minus "L00000000044$pb"
{standard input}:3483:symbol: "L_e_RetrieveError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:3425:non-relocatable subtraction expression, "L___stack_chk_guard$non_lazy_ptr" minus "L00000000043$pb"
{standard input}:3425:symbol: "L___stack_chk_guard$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:3405:non-relocatable subtraction expression, "L_e_RetrieveError$non_lazy_ptr" minus "L00000000043$pb"
{standard input}:3405:symbol: "L_e_RetrieveError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:3349:non-relocatable subtraction expression, "L_e_RetrieveError$non_lazy_ptr" minus "L00000000043$pb"
{standard input}:3349:symbol: "L_e_RetrieveError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:3330:non-relocatable subtraction expression, "L_rb_eTypeError$non_lazy_ptr" minus "L00000000043$pb"
{standard input}:3330:symbol: "L_rb_eTypeError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:3306:non-relocatable subtraction expression, "L___stack_chk_guard$non_lazy_ptr" minus "L00000000043$pb"
{standard input}:3306:symbol: "L___stack_chk_guard$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:3267:non-relocatable subtraction expression, "L_e_RetrieveError$non_lazy_ptr" minus "L00000000042$pb"
{standard input}:3267:symbol: "L_e_RetrieveError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:3206:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000041$pb"
{standard input}:3206:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:3147:non-relocatable subtraction expression, "L_rb_eArgError$non_lazy_ptr" minus "L00000000040$pb"
{standard input}:3147:symbol: "L_rb_eArgError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:3103:non-relocatable subtraction expression, "_c_domain_snapshot" minus "L00000000039$pb"
{standard input}:3103:symbol: "_c_domain_snapshot" can't be undefined in a subtraction expression
{standard input}:3070:non-relocatable subtraction expression, "L_rb_eSystemCallError$non_lazy_ptr" minus "L00000000038$pb"
{standard input}:3070:symbol: "L_rb_eSystemCallError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:3030:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000037$pb"
{standard input}:3030:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2981:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000036$pb"
{standard input}:2981:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2958:non-relocatable subtraction expression, "L_e_NoSupportError$non_lazy_ptr" minus "L00000000036$pb"
{standard input}:2958:symbol: "L_e_NoSupportError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2910:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000035$pb"
{standard input}:2910:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2887:non-relocatable subtraction expression, "L_e_NoSupportError$non_lazy_ptr" minus "L00000000035$pb"
{standard input}:2887:symbol: "L_e_NoSupportError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2839:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000034$pb"
{standard input}:2839:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2815:non-relocatable subtraction expression, "L_rb_eTypeError$non_lazy_ptr" minus "L00000000034$pb"
{standard input}:2815:symbol: "L_rb_eTypeError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2768:non-relocatable subtraction expression, "L_e_RetrieveError$non_lazy_ptr" minus "L00000000033$pb"
{standard input}:2768:symbol: "L_e_RetrieveError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2723:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000032$pb"
{standard input}:2723:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2705:non-relocatable subtraction expression, "L_e_NoSupportError$non_lazy_ptr" minus "L00000000032$pb"
{standard input}:2705:symbol: "L_e_NoSupportError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2659:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000031$pb"
{standard input}:2659:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2641:non-relocatable subtraction expression, "L_e_NoSupportError$non_lazy_ptr" minus "L00000000031$pb"
{standard input}:2641:symbol: "L_e_NoSupportError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2592:non-relocatable subtraction expression, "L_ruby_libvirt_str_new2_wrap$non_lazy_ptr" minus "L00000000030$pb"
{standard input}:2592:symbol: "L_ruby_libvirt_str_new2_wrap$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2574:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000030$pb"
{standard input}:2574:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2523:non-relocatable subtraction expression, "L___stack_chk_guard$non_lazy_ptr" minus "L00000000029$pb"
{standard input}:2523:symbol: "L___stack_chk_guard$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2510:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000029$pb"
{standard input}:2510:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2482:non-relocatable subtraction expression, "L_e_NoSupportError$non_lazy_ptr" minus "L00000000029$pb"
{standard input}:2482:symbol: "L_e_NoSupportError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2367:non-relocatable subtraction expression, "L___stack_chk_guard$non_lazy_ptr" minus "L00000000029$pb"
{standard input}:2367:symbol: "L___stack_chk_guard$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2330:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000028$pb"
{standard input}:2330:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2307:non-relocatable subtraction expression, "L_e_NoSupportError$non_lazy_ptr" minus "L00000000028$pb"
{standard input}:2307:symbol: "L_e_NoSupportError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2256:non-relocatable subtraction expression, "L_e_RetrieveError$non_lazy_ptr" minus "L00000000027$pb"
{standard input}:2256:symbol: "L_e_RetrieveError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2212:non-relocatable subtraction expression, "L_e_DefinitionError$non_lazy_ptr" minus "L00000000026$pb"
{standard input}:2212:symbol: "L_e_DefinitionError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2189:non-relocatable subtraction expression, "L_e_NoSupportError$non_lazy_ptr" minus "L00000000026$pb"
{standard input}:2189:symbol: "L_e_NoSupportError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2122:non-relocatable subtraction expression, "L_e_DefinitionError$non_lazy_ptr" minus "L00000000025$pb"
{standard input}:2122:symbol: "L_e_DefinitionError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2073:non-relocatable subtraction expression, "L_e_RetrieveError$non_lazy_ptr" minus "L00000000024$pb"
{standard input}:2073:symbol: "L_e_RetrieveError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2028:non-relocatable subtraction expression, "L_ruby_libvirt_str_new2_wrap$non_lazy_ptr" minus "L00000000023$pb"
{standard input}:2028:symbol: "L_ruby_libvirt_str_new2_wrap$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:2010:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000023$pb"
{standard input}:2010:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:1974:non-relocatable subtraction expression, "L___stack_chk_guard$non_lazy_ptr" minus "L00000000022$pb"
{standard input}:1974:symbol: "L___stack_chk_guard$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:1959:non-relocatable subtraction expression, "L_e_RetrieveError$non_lazy_ptr" minus "L00000000022$pb"
{standard input}:1959:symbol: "L_e_RetrieveError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:1941:non-relocatable subtraction expression, "L___stack_chk_guard$non_lazy_ptr" minus "L00000000022$pb"
{standard input}:1941:symbol: "L___stack_chk_guard$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:1907:non-relocatable subtraction expression, "L_e_RetrieveError$non_lazy_ptr" minus "L00000000021$pb"
{standard input}:1907:symbol: "L_e_RetrieveError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:1858:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000020$pb"
{standard input}:1858:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:1732:non-relocatable subtraction expression, "_c_domain_ifinfo" minus "L00000000019$pb"
{standard input}:1732:symbol: "_c_domain_ifinfo" can't be undefined in a subtraction expression
{standard input}:1720:non-relocatable subtraction expression, "L_e_RetrieveError$non_lazy_ptr" minus "L00000000019$pb"
{standard input}:1720:symbol: "L_e_RetrieveError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:1635:non-relocatable subtraction expression, "L___stack_chk_guard$non_lazy_ptr" minus "L00000000018$pb"
{standard input}:1635:symbol: "L___stack_chk_guard$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:1485:non-relocatable subtraction expression, "_c_domain_vcpuinfo" minus "L00000000018$pb"
{standard input}:1485:symbol: "_c_domain_vcpuinfo" can't be undefined in a subtraction expression
{standard input}:1467:non-relocatable subtraction expression, "L_e_RetrieveError$non_lazy_ptr" minus "L00000000018$pb"
{standard input}:1467:symbol: "L_e_RetrieveError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:1385:non-relocatable subtraction expression, "L_e_RetrieveError$non_lazy_ptr" minus "L00000000018$pb"
{standard input}:1385:symbol: "L_e_RetrieveError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:1366:non-relocatable subtraction expression, "L___stack_chk_guard$non_lazy_ptr" minus "L00000000018$pb"
{standard input}:1366:symbol: "L___stack_chk_guard$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:1299:non-relocatable subtraction expression, "_c_domain_block_info" minus "L00000000017$pb"
{standard input}:1299:symbol: "_c_domain_block_info" can't be undefined in a subtraction expression
{standard input}:1287:non-relocatable subtraction expression, "L_e_RetrieveError$non_lazy_ptr" minus "L00000000017$pb"
{standard input}:1287:symbol: "L_e_RetrieveError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:1171:non-relocatable subtraction expression, "_c_domain_memory_stats" minus "L00000000016$pb"
{standard input}:1171:symbol: "_c_domain_memory_stats" can't be undefined in a subtraction expression
{standard input}:1152:non-relocatable subtraction expression, "L_e_RetrieveError$non_lazy_ptr" minus "L00000000016$pb"
{standard input}:1152:symbol: "L_e_RetrieveError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:1038:non-relocatable subtraction expression, "_c_domain_block_stats" minus "L00000000015$pb"
{standard input}:1038:symbol: "_c_domain_block_stats" can't be undefined in a subtraction expression
{standard input}:1026:non-relocatable subtraction expression, "L_e_RetrieveError$non_lazy_ptr" minus "L00000000015$pb"
{standard input}:1026:symbol: "L_e_RetrieveError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:901:non-relocatable subtraction expression, "_c_domain_info" minus "L00000000014$pb"
{standard input}:901:symbol: "_c_domain_info" can't be undefined in a subtraction expression
{standard input}:889:non-relocatable subtraction expression, "L_e_RetrieveError$non_lazy_ptr" minus "L00000000014$pb"
{standard input}:889:symbol: "L_e_RetrieveError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:834:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000013$pb"
{standard input}:834:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:786:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000012$pb"
{standard input}:786:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:720:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000011$pb"
{standard input}:720:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:697:non-relocatable subtraction expression, "L_e_NoSupportError$non_lazy_ptr" minus "L00000000011$pb"
{standard input}:697:symbol: "L_e_NoSupportError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:684:non-relocatable subtraction expression, "L_e_NoSupportError$non_lazy_ptr" minus "L00000000011$pb"
{standard input}:684:symbol: "L_e_NoSupportError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:630:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000010$pb"
{standard input}:630:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:588:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000009$pb"
{standard input}:588:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:546:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000008$pb"
{standard input}:546:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:528:non-relocatable subtraction expression, "L_e_NoSupportError$non_lazy_ptr" minus "L00000000008$pb"
{standard input}:528:symbol: "L_e_NoSupportError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:482:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000007$pb"
{standard input}:482:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:426:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000006$pb"
{standard input}:426:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:408:non-relocatable subtraction expression, "L_e_NoSupportError$non_lazy_ptr" minus "L00000000006$pb"
{standard input}:408:symbol: "L_e_NoSupportError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:353:non-relocatable subtraction expression, "L_e_Error$non_lazy_ptr" minus "L00000000005$pb"
{standard input}:353:symbol: "L_e_Error$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:192:non-relocatable subtraction expression, "L_rb_eTypeError$non_lazy_ptr" minus "L00000000004$pb"
{standard input}:192:symbol: "L_rb_eTypeError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:166:non-relocatable subtraction expression, "L_rb_eArgError$non_lazy_ptr" minus "L00000000004$pb"
{standard input}:166:symbol: "L_rb_eArgError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:79:non-relocatable subtraction expression, "L_rb_eArgError$non_lazy_ptr" minus "L00000000003$pb"
{standard input}:79:symbol: "L_rb_eArgError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:42:non-relocatable subtraction expression, "_c_domain" minus "L00000000002$pb"
{standard input}:42:symbol: "_c_domain" can't be undefined in a subtraction expression
{standard input}:20:non-relocatable subtraction expression, "L_rb_eSystemCallError$non_lazy_ptr" minus "L00000000001$pb"
{standard input}:20:symbol: "L_rb_eSystemCallError$non_lazy_ptr" can't be undefined in a subtraction expression
{standard input}:unknown:Undefined local symbol L_rb_eSystemCallError$non_lazy_ptr
{standard input}:unknown:Undefined local symbol L_rb_eArgError$non_lazy_ptr
{standard input}:unknown:Undefined local symbol L_rb_eTypeError$non_lazy_ptr
{standard input}:unknown:Undefined local symbol L_e_Error$non_lazy_ptr
{standard input}:unknown:Undefined local symbol L_e_NoSupportError$non_lazy_ptr
{standard input}:unknown:Undefined local symbol L_e_RetrieveError$non_lazy_ptr
{standard input}:unknown:Undefined local symbol L___stack_chk_guard$non_lazy_ptr
{standard input}:unknown:Undefined local symbol L_ruby_libvirt_str_new2_wrap$non_lazy_ptr
{standard input}:unknown:Undefined local symbol L_e_DefinitionError$non_lazy_ptr
fatal error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lipo: can't open input file: /var/folders/fg/20n5jbdn60s7l3pgmnnz1htw0000gn/T//cc8Ld1Qx.out (No such file or directory)
make: *** [domain.o] Error 1


Gem files will remain installed in /Users/ryanfleisher/.vagrant.d/gems/gems/ruby-libvirt-0.5.2 for inspection.
Results logged to /Users/ryanfleisher/.vagrant.d/gems/gems/ruby-libvirt-0.5.2/ext/libvirt/gem_make.out
```

**Comment by Trefex** [comment]
Same problem here.

Did you ever figure this out?

**Comment by soulthani** [comment]
Hi

based on this thread : https://github.com/pradels/vagrant-libvirt/issues/497

heres what I do : 

1. when doing vagrant plugin install vagrant-libivirt, we got these 2 following line information 
Gem files will remain installed in /Users/sultan/.vagrant.d/gems/gems/ruby-libvirt-0.6.0 for inspection.
Results logged to /Users/sultan/.vagrant.d/gems/extensions/universal-darwin-12/2.2.0/ruby-libvirt-0.6.0/gem_make.out

I copy  /Users/sultan/.vagrant.d/gems/gems/ruby-libvirt-0.6.0 to ~/.

2. vi ext/libvirt/extconf.rb, add [this changes](https://github.com/kontrafiktion/ruby-libvirt/commit/3f768dc492353a143b45c08180fcd690885c2b09)  to this file

3. $ /opt/vagrant/embedded/bin/rake package
4. $ /opt/vagrant/embedded/bin/gem install -i ~/.vagrant.d/gems/ --local pkg/ruby-libvirt-0.6.0.gem

Building native extensions.  This could take a while...
Successfully installed ruby-libvirt-0.6.0
Parsing documentation for ruby-libvirt-0.6.0
Installing ri documentation for ruby-libvirt-0.6.0
Done installing documentation for ruby-libvirt after 18 seconds
1 gem installed

5. then install the plugin 
$ vagrant plugin install vagrant-libvirt
Installing the 'vagrant-libvirt' plugin. This can take a few minutes...
Installed the plugin 'vagrant-libvirt (0.0.33)'!

**Comment by soulthani** [comment]
Hi

based on this thread : https://github.com/pradels/vagrant-libvirt/issues/497

heres what I do : 

1> when doing vagrant plugin install vagrant-libivirt, we got these 2 following line information 
	

> Gem files will remain installed in /Users/sultan/.vagrant.d/gems/gems/ruby-libvirt-0.6.0 for inspection.
> Results logged to /Users/sultan/.vagrant.d/gems/extensions/universal-darwin-12/2.2.0/ruby-libvirt-0.6.0/gem_make.out

I copy  /Users/sultan/.vagrant.d/gems/gems/ruby-libvirt-0.6.0 to ~/.

2> vi ext/libvirt/extconf.rb, add [this changes](https://github.com/kontrafiktion/ruby-libvirt/commit/3f768dc492353a143b45c08180fcd690885c2b09)  to this file

3> $ /opt/vagrant/embedded/bin/rake package
4> $ /opt/vagrant/embedded/bin/gem install -i ~/.vagrant.d/gems/ --local pkg/ruby-libvirt-0.6.0.gem

Building native extensions.  This could take a while...
Successfully installed ruby-libvirt-0.6.0
Parsing documentation for ruby-libvirt-0.6.0
Installing ri documentation for ruby-libvirt-0.6.0
Done installing documentation for ruby-libvirt after 18 seconds
1 gem installed

5> then install the plugin 
$ vagrant plugin install vagrant-libvirt
Installing the 'vagrant-libvirt' plugin. This can take a few minutes...
Installed the plugin 'vagrant-libvirt (0.0.33)'!

**Comment by infernix** [comment]
Please do report this to ruby-libvirt as we cannot resolve the issue in vagrant-libvirt. I'm also closing this for now as a duplicate of #497 since the discussion seems to be happening mostly there.
